data/pc/1.10/biomes.json

```diff
@@ -21,7 +21,8 @@
     "dimension": "overworld",
     "displayName": "Plains",
     "color": 9286496,
-    "rainfall": 0.4
+    "rainfall": 0.4,
+    "child": 129
```

Comment:
Can there be more then one child for biom variations?

Reply:
No only one